src/screens/login/login-page.tsx

```diff
@@ -1,23 +1,56 @@
-import React from 'react';
-import {  View,  Text, TouchableOpacity, TextInput } from 'react-native';
-import { styles } from './login-page.styles';
+import React, {useState} from 'react';
+import {Text, TextInput, TouchableOpacity, View} from 'react-native';
+import {styles} from './login-page.styles';
+import {
+  emailIsValid,
+  isEmpty,
+  passwordHasLetter,
+  passwordHasNumber,
+  passwordHasLength,
+} from './login-validation';
 
-export function LoginPage(){
-    return(
-        <View style={styles.container}>
-            <Text style={styles.header}>Bem-vindo(a) à Taqtile!</Text>
+export const LoginPage = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [errorMessage, setErrorMessage] = useState('');
 
-            <View style={styles.wrapperInput}>
-                <Text style={styles.label}>E-mail</Text>
-                <TextInput style={styles.input} />
-                <Text style={styles.label}>Senha</Text>
-                <TextInput style={styles.input} />
-            </View>
+  const loginValidation = () => {
+    if (isEmpty(email, password)) {
```

Comment:
```suggestion
    if (isEmpty(email) || isEmpty(password)) {
```
e no `login-validation.tsx` (a propósito, esse arquivo `login-validation.tsx` não precisa ser .tsx. só precisa ser .tsx arquivos que tem componentes React 👍 )
```ts
export const isEmpty = (value: string) => !value;
```